charts/opentelemetry-demo/templates/NOTES.txt

```diff
@@ -19,14 +19,3 @@
   Feature Flags UI     http://localhost:8080/feature/
   Load Generator UI    http://localhost:8080/loadgen/
   Jaeger UI            http://localhost:8080/jaeger/ui/
-
-- OpenTelemetry Collector OTLP/HTTP receiver (required for browser spans to be emitted):
-  by running these commands:
-     kubectl port-forward svc/{{ include "otel-demo.name" . }}-otelcol 4318:4318
-
-{{- if eq (index .Values "opentelemetry-collector" "config" "receivers" "otlp" "protocols" "http" "endpoint") "0.0.0.0:4318" }}
-
-The OpenTelemetry Collector OTLP/HTTP receiver is bound to 0.0.0.0 in order to allow the "kubectl port-forward" command.
-This may be susceptible to denial of service attacks.
-See: CWE-1327 https://cwe.mitre.org/data/definitions/1327.html
```

Comment:
What about demo 1.5 lets us remove this info?

Reply:
We no longer need to expose the collector via `kubectl port-forward`, because we have a route built into the frontendproxy to route the traffic.

And with that comment, I realized we updated the notes but not the actual binding of the collector endpoints... update to this PR incoming.

Reply:
I updated the Prometheus bind endpoint to use the NODE IP instead.